.github/workflows/test_cli.yml

```diff
@@ -15,12 +15,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python: ["3.7", "3.8", "3.9", "3.10"]
-        dbt: ["1.0.*", "1.1.*", "1.2.*"]
-        exclude:
-          # Only DBT 1.1+ supports Python 3.10 (officially)
-          - python: 3.10
-            dbt: "1.0.*"
+        python:
+          - "3.7"
+          # - "3.8"
+          # - "3.9"
```

Comment:
We should uncomment these?

Reply:
I did this on purpose because if it works for 3.7 and 3.10, it probably works for the ones in the middle. Would you agree?

Reply:
3.8 and 3.9 are probably the most popular versions right now, so we should definitely test on those.

Reply:
Maybe at least 3.8.